modules/bitgo/src/v2/wallet.ts

```diff
@@ -663,6 +665,15 @@ export class Wallet {
     return this._wallet.spendableBalanceString;
   }
 
+  /**
+   * Get a string representation of the TRS balance of this wallet
+   *
+   * This is useful when balances have the potential to overflow standard javascript numbers
+   */
+  trsBalanceString(): string {
```

Comment:
I'm not sure how this solves the problem in the ticket. Probably missing some context. Can you expand on how these new fields are utilized with an example?

Reply:
Sure, these wallet objects are passed back to bga when executing the wallet command and then used to display the result:
https://github.com/BitGo/bitgo-admin/blob/master/src/handlers/wallet.js#L151

I wanted to add the property to the object instead of doing a direct property reference on a undeclared field. The TRS balance field will go right below the other 3 balances that get printed.

This PR will work in tandem with 2 other PRs to achieve this. Here is the PR that will populate that new field:
https://github.com/BitGo/bitgo-microservices/pull/18262/files#diff-008ef6124f852724918acc1cb7c461bd3409c7afae62c2740b9952e00f81fd94R417

Reply:
To make sure, these are fields that only for bga?